views/index.erb

```diff
@@ -0,0 +1,70 @@
+<link rel="stylesheet" href="peep.css">
```

Comment:
This erb file was really clear to read, I could understand everything that would be displayed from the if statements. I like the content wording as well. I can imagine it looks great in the browser!